static/pages/volunteer/Volunteer.js

```diff
@@ -12,8 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-oppiaFoundationWebsite.controller('VolunteerPage', ['$scope', '$document',
-  '$window', function($scope, $document, $window) {
+oppiaFoundationWebsite.controller('VolunteerPage', [
+  '$scope', '$document', '$http', '$mdDialog', '$log', '$window',
+  'MAILHANDLER_URL', 'VOLUNTEER_EMAIL_TYPE', 'THANKYOU_MESSAGE',
```

Comment:
Line breaks below should match those here, for easy verification that the dependency lists match.

Reply:
Done!